Rakefile

```diff
@@ -22,3 +24,8 @@ load 'rails/tasks/statistics.rake'
 Bundler::GemHelper.install_tasks
 
 Dir['lib/tasks/*.rake'].each { |r| load r }
+
+spec = Gem::Specification.find_by_name 'bower-rails'
+load "#{spec.gem_dir}/lib/tasks/bower.rake"
```

Comment:
It's one of those things that are a bit weird with ruby...

```bash
niko@pro4:~/dev/backoffice (master) $ be irb
irb(main):001:0> puts $LOAD_PATH
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/wicked-1.2.1/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/webmock-1.24.1/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/virtus-1.0.5/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/vcr-3.0.1/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/user_impersonate2-0.10.2/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/bundler/gems/ui_components-4921072030b2/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/underscore-rails-1.8.3/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/uglifier-2.7.2/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/timecop-0.8.0/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/teaspoon-jasmine-2.3.4/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/teaspoon-1.1.4/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/sprockets-es6-0.9.0/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-commands-rspec-1.0.4/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/soapy_yandex-0.0.4/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/soapy_cake-1.22.1/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/slim-rails-3.0.1/lib
/Users/niko/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/simplecov-0.11.2/lib
[...]
```

(notice there is no rails or autoloading in this example!)